docker/Dockerfile

```diff
@@ -43,11 +45,10 @@ RUN git clone -q https://github.com/kthohr/stats \
 # install ACE
 WORKDIR /opt
 
-ADD https://api.github.com/repos/SystemsGenetics/ACE/git/refs/heads/${ACE_REVISION} ace-version.json
```

Comment:
Why was this line removed? It allows you to push commits to ACE and then rebuild the docker image with the new commits. Otherwise docker would cache the previous git clone command and not pull the new commits. ALso the line below for cloning the KINC repo.

Reply:
It doesn't work for tagged releases, which makes sense as those don't change.  I can try and wrap it in an if statement in case the revision is 'develop'.

Reply:
It seems you can't use an IF statement for an ADD statement.  I'm not sure how to fix this one.

Reply:
Okay now I remember for tagged releases you have to use the URL `refs/tags` instead of `refs/heads`. So you can add a conditional for that, or just leave it as is if you don't want to deal with it. Not a critical feature, just makes things easier if you only updated the github repo before a docker build.

Reply:
I don't think it's possible to add a conditional for ADD statements :-(  But I'm happy to do it if you know how. I couldn't find a way to do it.